File: src/main/java/com/example/healthylifestyleapp/ProfileScreenActivity.java
package com.example.healthylifestyleapp;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;
import android.widget.RelativeLayout;

import androidx.appcompat.app.AppCompatActivity;

public class ProfileScreenActivity extends AppCompatActivity implements View.OnClickListener {
    ImageView ImageViewBackProfile;
RelativeLayout rlHeaderFitnessDevice;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_profile_screen);
        ImageViewBackProfile=findViewById(R.id.ImageViewBackProfile);
        ImageViewBackProfile.setOnClickListener(this);
        rlHeaderFitnessDevice=findViewById(R.id.rlHeaderFitnessDevice);
        rlHeaderFitnessDevice.setOnClickListener(this);

    }

    @Override
    public void onClick(View v) {
        switch (v.getId())
        {
            case R.id.ImageViewBackProfile:
                Intent intentBack= new Intent(this,SettingsActivity.class);
                startActivity(intentBack);
                break;
            case R.id.rlHeaderFitnessDevice:
                Intent fitnessIntent= new Intent(this,FitnessDevice.class);
                startActivity(fitnessIntent);
                break;

        }
    }
}

File: app/src/main/java/com/example/healthylifestyleapp/ui/activities/ProfileScreenActivity.kt
package com.example.healthylifestyleapp.ui.activities

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.example.healthylifestyleapp.R
import kotlinx.android.synthetic.main.activity_profile_screen.*

class ProfileScreenActivity : AppCompatActivity(), View.OnClickListener {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_profile_screen)
        rlHeaderFitnessDevice.setOnClickListener(this)

    }

    override fun onClick(v: View) {
        when (v.id) {
            R.id.rlHeaderFitnessDevice -> {
                val fitnessIntent = Intent(this, FitnessDevice::class.java)
                startActivity(fitnessIntent)
            }
        }
    }
}
